webpack.config.js

```diff
@@ -53,6 +53,13 @@ const layouts = [
   }
 ]
 
+const addons = [{
+  from: 'addons/**/*',
+  to: TARGET,
+  // strip addon folder, copy everything over
+  transformPath: (path) => path.replace(/^addons\/([^\/]*)\//,"")
```

Comment:
```suggestion
  transformPath: (path) => path.replace(/^addons\/([^\/]*)\//, ENV === 'production' ? '' : 'elements/')
```

And add `...addons` to dev mode to so we can easilly load addons in index.html (ie elements/nuxeo-dam.html)

Reply:
But does this mean we'll need a server with all addon installed to have the web ui behaving properly?

Reply:
no, it's just for convenience, in case you want to debug nuxeo-dam you can just add `elements/nuxeo-dam.html` to the list of bundles in your index.html